vars: add placeholder session variable for xmloption

otan opened the pull request:
Release note (sql change): The `xmloption` session variable is now
accepted, only taking in `content`. Note this does not do anything.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/74664)
<!-- Reviewable:end -->

otan committed:
vars: add placeholder session variable for xmloption

Release note (sql change): The `xmloption` session variable is now
accepted, only taking in `content`. Note this does not do anything.

otan commented:
thanks

bors r=rafiss
